frontend/src/metabase/visualizations/components/ScalarValue.styled.tsx

```diff
@@ -0,0 +1,9 @@
+import styled from "styled-components";
+
+export const ScalarRoot = styled.div`
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+`;
```

Comment:
Should this also mirror `flex-wrap` and `relative` classes?

Reply:
Nice catch!